nlp/2021_06_18_data_augmentation/totw_nlp_dat_aug.ipynb

```diff
@@ -0,0 +1,1789 @@
+          }
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n",
+            "\n",
+            "\n",
+            "Training completed. Do not forget to share your model on huggingface.co/models =)\n",
+            "\n",
+            "\n",
+            "***** Running Evaluation *****\n",
+            "  Num examples = 617\n",
+            "  Batch size = 8\n"
+          ],
+          "name": "stderr"
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/html": [
+              "\n",
+              "    <div>\n",
+              "      \n",
+              "      <progress value='78' max='78' style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
+              "      [78/78 00:02]\n",
+              "    </div>\n",
+              "    "
+            ],
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yoGbjDeBvL9t"
+      },
+      "source": [
+        "### Model with combined augmented datasets, the whole merry gang together"
```

Comment:
nitpick: this section didn't seem to hare run. Meaning that in the rendered notebook, the others have clear output which shows the accuracy, but this one doesn't

Reply:
Typo brainfart from my end: meant to say "didn't seem to have a run" :)